bin/common/utils.js

```diff
@@ -32,9 +31,20 @@ const getLatestHost = (argv) => {
   return { port, host, socket }
 }
 
+exports.runAsyncScript = async (fn) => {
+  try {
+    await fn()
+    process.exit(0)
+  } catch (err) {
+    consola.fatal('Failed to run async Nuxt script!')
+    consola.fatal(err)
+    process.exit(1)
```

Comment:
Is it required? Is even second fatal executed? There is a hook to exit on fatal
https://github.com/nuxt/nuxt.js/blob/dev/bin/nuxt#L12-L19

Reply:
Good catch!